Sources/TokamakCore/StackReconciler.swift

```diff
@@ -58,6 +58,12 @@ public final class StackReconciler<R: Renderer> {
    */
   public let rootTarget: R.TargetType
 
+  /** A root renderer's main preference store
```

Comment:
nit: comment formatting 
```suggestion
  /** A root renderer's main preference store.
```